### PR TITLE
Skip indentation of empty lines when indenting a selection.

### LINKED ISSUE #41113: Blank lines are indented when indenting multi-line selections
**Godot version:**
3.2.2 stable (previous versions, and master branch)

**Issue description:**
When indenting a text selection, empty lines will also be indented, creating uneven indentation in almost every use-case where the selected text was functional code.

I find this cumbersome when refactoring or repurposing code and haven't been able to come up with a use-case where it might be valuable.

**Steps to reproduce:**
1. Select a block of code that has a mixture of text and empty lines
2. Press Tab to indent
3. Note uneven indentation (line 5 in the attached recording)

Unwanted indentation

### PR DESCRIPTION
**Pull request by Cheeseness**
Closes #41113

### CODE REVIEW

File: scene/gui/text_edit.cpp [hunk 1]
```diff
@@ -1710,6 +1710,8 @@ void TextEdit::indent_right() {
 
 	for (int i = start_line; i <= end_line; i++) {
 		String line_text = get_line(i);
+		if (line_text.size() == 0 && is_selection_active())
+			continue;
```
Comment (akien-mga) [inline scene/gui/text_edit.cpp:1714]:
```suggestion
		if (line_text.size() == 0 && is_selection_active()) {
			continue;
		}
```

Comment (Cheeseness) [reply scene/gui/text_edit.cpp]: Done, and rebased. Thanks!

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.3.

**Pull request merged**